src/LiveComponent/src/Twig/LiveComponentExtension.php

```diff
@@ -25,6 +25,7 @@ public function getFunctions(): array
     {
         return [
             new TwigFunction('component_url', [LiveComponentRuntime::class, 'getComponentUrl']),
+            new TwigFunction('live_action', [LiveComponentRuntime::class, 'getLiveAction'], ['is_safe' => ['html_attr']]),
```

Comment:
```suggestion
            new TwigFunction('live_action', [LiveComponentRuntime::class, 'liveAction'], ['is_safe' => ['html_attr']]),
```

Feels a bit weird to use '_get_' here, as if we returned the Attribute instance... maybe `liveAction` or `renderLiveAction` to be precise ?